src/entities/elastic-storage/utxo-data.js

```diff
@@ -12,7 +12,15 @@ export type UtxoType = {
   amount: number,
 }
 
-export const getTxInputUtxoId = (input: TxInputType) => `${UTXO_OUTPUT_TYPE}:${input.txId}:${input.idx}`
+export const getTxInputUtxoId = (input: TxInputType) => {
+  switch (input.type) {
+    case 'utxo':
+      return `${UTXO_OUTPUT_TYPE}:${input.txId}:${input.idx}`
```

Comment:
Why do we have this different from the old UtxoId of `${input.txId}${input.idx}`?

Reply:
Ah, it's because in Elastic we are storing all **inputs** and **outputs** together in a separate index `txio` (tx-inputs/outputs), so they require separate IDs (e.g. `input:tx1234:0` and `output:tx1234:0`). This might look strange from the protocol level where inputs are just links to previously existing outputs, but from the point of Elastic being NoSQL document-based by itself and also being used for an explorer - it gives us the ability to query these entities by other relevant field, and they need to have unique IDs to be in a single index.